backend/templates/centered.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load cms_tags%}
+
+{% block title %}
```

Comment:
What's the purpose of this block? I'm struggling to understand how it works. :)

Reply:
The first line ensures that this template inherits the structure of "base.html"
The cms_tag need to be loaded again nonetheless according to a tutorial I followed. Removing that line results in errors since the following cms tags are not understood. 

The 4th line initiates a block for title display (up in the browser tab i believe) but apparently it is not necessary since this function is also inherited from base.html as I just tested. I will remove it from all templates but base.html